schemas/README.md

```diff
@@ -0,0 +1,98 @@
+# Schemas of OpenChain Telco Guide
+This directory is an implementation of json schemas of Telco Guide. The file [openchain-telco-guide-v1.0-schema.json](openchain-telco-guide-v1.0-schema.json) is just the json schema file you can take to your project. 
+
+As further described below, a technology viewpoint is shown below how to generate json schema file from scratch if you wanna a clear comprehension of underlying logic.
+
+## SPDX 2.2 and SPDX 2.3
+In the specification of OpenChain Telco Guide, we say 'An OpenChain Telco SBOM Guide compatible document SHALL adhere to the version 2.2 of the SPDX Data Format as standardized in ISO/IEC 5962:2021, or to the version 2.3 of the standard'. By 05/08/2024, v2.2, v2.2.1, v2.2.2 and v2.3 have been released in github website. We choose v2.2.2 and v2.3 as origin json schema implementation.
```

Comment:
I would avoid date formats like "05/08/2024". My personal preference is ISO 8061 yyyy-mm-dd which tends to have the least opportunities for misinterpretations. Wiki: https://en.wikipedia.org/wiki/ISO_8601
When using "05/08/2024" it is difficult to say if we are talking about May 8th or August 5th.

Reply:
Thanks for your review. I will modify it later.